src/game/GameResults.as

```diff
@@ -894,6 +904,9 @@ package game
             // Get last score
             var gameResult:GameScoreResult = songResults[songResults.length - 1];
 
+            // Update Judge Offset
+            updateJudgeOffset(gameResult);
```

Comment:
What did this fix?

Reply:
Offset was only being saved on song score success, that isn't really a requirement for that function and is better placed elsewhere.